### PR TITLE
feat(MultiTwitch): add presence

### PR DESCRIPTION
**Pull request by sweezyio**
## Description 
Presence for [MultiTwitch](https://multitwitch.tv), a platform allowing users to watch any number of Twitch streams simultaneously all on the same page. 

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
- Side-by-Side comparison
- Profile Image 1
- Profile Image 2
</details>

### CODE REVIEW

**Comment by RisingSunLight42** [comment]
Revert your merge and fix your pull request description

**Review by RisingSunLight42 — COMMENTED** [review]
Really low quality I think. You could add a slideshow to display every streams on Discord to make the presence more interesting.

File: websites/M/MultiTwitch/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "181381194070491136",
+		"name": "3Tau#7777"
```
Comment (Bas950) [inline websites/M/MultiTwitch/metadata.json:5]:
```suggestion
		"name": "3Tau"
```

File: websites/M/MultiTwitch/metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	},
+	"service": "MultiTwitch",
+	"description": {
+		"en": "Watch and enjoy an unlimited number of your favorite Twitch streams simultaneously."
+	},
+	"url": [
+		"multitwitch.tv",
+		"www.multitwitch.tv"
+	],
+	"regExp": "(www.)?(multitwitch.tv)",
```
Comment (Bas950) [inline websites/M/MultiTwitch/metadata.json:15]:
```suggestion
```
Seems like a useless regex, the url should be enough

**Review by Bas950 — CHANGES_REQUESTED** [review]
Could you expand it to show more data/pages? Cause it would be denied for low quality.

**Comment by sweezyio** [comment]
I updated the presence to use a slideshow to display all of the streams that are currently being watched (and an option to disable that in favor of a simple timestamp). And I added the option to include a button in the presence to open the current URL from Discord.

Updated presence with settings:
premid settings

**Review by sweezyio — COMMENTED** [review]
Updated to show more detail

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**